end-to-end-tests/package.json

```diff
@@ -6,13 +6,14 @@
   "scripts": {
     "test-travis": "rm -rf screenshots/diff/; wdio wdio/travis.conf.js || bash image-compare/upload_failed_screenshots.sh",
     "test-webdriver-manager": "rm -rf screenshots/diff/; cd .. && ./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cd end-to-end-tests && (wdio wdio/webdriver-manager.conf.js || bash image-compare/upload_failed_screenshots.sh)",
-    "e2e-local": "rm -rf screenshots/screen/; rm -rf screenshots/reference/; rm -rf screenshots/diff/; cd .. && ./scripts/env_vars.sh && eval \"$(./scripts/env_vars.sh)\" && cd end-to-end-tests && (wdio wdio/webdriver-manager.conf.js || bash image-compare/upload_failed_screenshots.sh)"
-
+    "e2e": "yarn run clear && yarn run test-webdriver-manager",
+    "clear": "rm -rf screenshots/screen/; rm -rf screenshots/reference/; rm -rf errorShots/; rm -rf screenshots/diff/;"
```

Comment:
whats the purpose of this?

Reply:
when you run tests locally, the screenshots are going to diff anyway, so it's sort of pointless to ever do a comparison against the real reference.

Reply:
ah okay

Reply:
while we're here do you want to delete `test-travis`

Reply:
sure